mentoring/light_children.py

```diff
@@ -101,12 +101,17 @@ def _is_default(value):
 
     @classmethod
     def init_block_from_node(cls, block, node, attr):
+        from mentoring import MentoringBlock
         block.light_children = []
         for child_id, xml_child in enumerate(node):
             cls.add_node_as_child(block, xml_child, child_id)
 
         for name, value in attr:
-            setattr(block, name, value)
+            if isinstance(block, MentoringBlock):
```

Comment:
Minor given the other comment below, but this file shouldn't contain anything specific to Mentoring at this level -- if you want to check whether this is a real XBlock, you should check for the XBlock class, not MentoringBlock.